sessions/css-basics/shapes/readme.md

```diff
@@ -0,0 +1,11 @@
+There are 4 divs to style for this exercise:
+
+1. div should be a square
+2. div should have rounded edges
+3. div should be a circle
+4. div should be rotated 45deg
+
+Also:
+
+- Use the classes that are already attached to the elements
+- Every div should have a unique color.
```

Comment:
A suggestion to add a bit more structure to the readme. And also, I would probably rename the file to `README.md`.
```suggestion
# CSS Basics - Different shapes

There are 4 `div` elements to style for this exercise:

1. `div` should be a square
2. `div` should have rounded edges
3. `div` should be a circle
4. `div` should be rotated 45deg

Also:

- Use the classes that are already attached to the elements
- Each div should have a unique color.
```